src/stores/IngestStore.js

```diff
@@ -422,6 +445,25 @@ class IngestStore {
         metadataSubtree: UrlJoin("production_master", "variants", "default", "streams")
       }));
 
+      let missingStreams = [];
+      if(!streams.audio) { missingStreams.push("audio"); }
+      if(!streams.video) { missingStreams.push("video"); }
+
+      if(missingStreams.length > 0) {
+        const response = yield this.ShowWarningDialog({
+          title: "Streams Not Found",
+          description: `No ${missingStreams.join(", ")} streams found in the media file. Would you like to continue the ingest?`
```

Comment:
Change to "No suitable ... stream(s) found..."

(The server-side heuristic for choosing streams can find an audio stream but decide not use it because e.g. it had an unsupported audio channel count.)